### PR TITLE
Mute proxy

### PR DESCRIPTION
**Pull request by dbadura**
**Description**

Changes proposed in this pull request:

- Mute istio-proxy
https://istio.io/latest/docs/tasks/observability/logs/access-log/

**Related issue(s)**

### CODE REVIEW

File: resources/istio/files/istio-operator-cluster-evaluation.yaml [hunk 1]
```diff
@@ -172,7 +172,7 @@ spec:
             maxSurge: 100%
             maxUnavailable: 25%
   meshConfig:
-    accessLogFile: /dev/stdout
+    accessLogFile: ""
```
Comment (dbadura) [inline resources/istio/files/istio-operator-cluster-evaluation.yaml:175]: I can add docs about it in comment, what do u think?

Comment (colunira) [reply resources/istio/files/istio-operator-cluster-evaluation.yaml:175]: That's a good idea. Please add it in every file, so nobody forgets about it :)

**Review by strekm — APPROVED** [review]
/meow

**Pull request merged**